src/discord/interaction_handlers/application_commands.rs

```diff
@@ -1308,6 +1313,49 @@ async fn handle_create_season(
     )))
 }
 
+async fn handle_see_unscheduled_races(
+    _ac: Box<CommandData>,
+    state: &Arc<DiscordState>,
+) -> Result<InteractionResponse, String> {
+    let mut cxn = state.diesel_cxn().await.map_err(|e| e.to_string())?;
+
+    let mut unscheduled = BracketRace::unscheduled(cxn.deref_mut()).map_err_to_string()?;
+    unscheduled.sort_by_key(|br| br.bracket_id);
+    let mut fields = vec![];
+    for br in unscheduled {
+        let (p1, p2) = br.players(cxn.deref_mut()).map_err_to_string()?;
+        let bname = br.bracket(cxn.deref_mut()).map_err_to_string()?.name;
```

Comment:
this is 2 queries per race, which is a lot of extra queries. it doesn't really matter - this is a sqlite database of ~150 rows - but it feels dumb anyway